public/members.json

```diff
@@ -1 +1 @@
-[{"member":"Googol","url":"https://googol.now.sh/"},{"member":"Caleb Denio","url":"https://calebdenio.me/"},{"member":"Rafael Cenzano","url":"https://rafaelcenzano.com/"},{"member":"Adrian Lobo","url":"https://loboadrian.github.io/"},{"member":"Hemesh","url":"https://hemesh.tech/"},{"member":"Ong Zhi Zheng","url":"https://ongzz.me/"},{"member":"Anthony Kung","url":"https://anthonykung.com/"},{"member":"Pragya Rajpurohit","url":"https://pragyaraj.tech/"},{"member":"Cole Wilson","url":"https://colewilson.xyz/"},{"member":"Rishi Kothari","url":"https://rishi.cx/"},{"member":"Khushraj Rathod","url":"https://www.khushrajrathod.com/"},{"member":"Ali A. Saleh","url":"https://Saleh.HackClub.com/"},{"member":"John Lins","url":"https://www.johnlins.com/"},{"member":"Benjamin Smith","url":"https://benjaminsmith.dev"},{"member":"Tejas Agarwal","url":"https://tejasagarwal.tech/"},{"member":"Vusal Ismayilov","url":"https://woosal.com"}]
\ No newline at end of file
+[{"member":"Googol","url":"https://googol.now.sh/"},{"member":"Caleb Denio","url":"https://calebdenio.me/"},{"member":"Rafael Cenzano","url":"https://rafaelcenzano.com/"},{"member":"Adrian Lobo","url":"https://loboadrian.github.io/"},{"member":"Hemesh","url":"https://hemesh.tech/"},{"member":"Ong Zhi Zheng","url":"https://ongzz.me/"},{"member":"Anthony Kung","url":"https://anthonykung.com/"},{"member":"Pragya Rajpurohit","url":"https://pragyaraj.tech/"},{"member":"Cole Wilson","url":"https://colewilson.xyz/"},{"member":"Rishi Kothari","url":"https://rishi.cx/","bypass":true},{"member":"Khushraj Rathod","url":"https://www.khushrajrathod.com/"},{"member":"Ali A. Saleh","url":"https://Saleh.HackClub.com/"},{"member":"John Lins","url":"https://www.johnlins.com/"},{"member":"Benjamin Smith","url":"https://benjaminsmith.dev"},{"member":"Tejas Agarwal","url":"https://tejasagarwal.tech/"},{"member":"Vusal Ismayilov","url":"https://woosal.com"}]
```

Comment:
We need to remove Rishi

Reply:
Ok, then at that point, I suppose the commit for that can be reverted, or should I keep that?

Reply:
lets keep support for the bypass but remove it from Rishi

Reply:
done!

Reply:
Ok, I think it's good now!